sycl/test/extensions/properties/properties_kernel_device_has.cpp

```diff
@@ -37,90 +36,65 @@ int main() {
   auto Redu1 = reduction<int>(nullptr, plus<int>());
   auto Redu2 = reduction<float>(nullptr, multiplies<float>());
 
-  // CHECK-IR: spir_kernel void @{{.*}}WGSizeKernel0(){{.*}} #[[DHAttr1:[0-9]+]]
```

Comment:
These checks (as well as the tests below) were removed as no longer relevant. This functionality is tested in `llvm/tests/SYCLLowerIR/PropagateAspectsUsage/*` , yet I'm as bit concerned by the change I make. If you have suggestions on how to keep the tests - I'm open for proposals.

Reply:
Yeah I'm a bit worried with the test removals. My only idea is to add a hidden -mllvm option or something to run the pass or not.

Reply:
Well, unfortunately it won't work as opt doesn't provide a way to exclude one pass from the pipeline.

Reply:
> Well, unfortunately it won't work as opt doesn't provide a way to exclude one pass from the pipeline.

The pass can skip itself if an option is passed. `BackendUtil.cpp` can react to an option to avoid adding a pass into the pipeline at all.

Tagging @steffenlarsen to review the test, he is likely an author of it.

Reply:
I ended up adding -Xclang -disable-llvm-passes to the tests as Alexey has suggested below